tests/test_utils.py

```diff
@@ -45,3 +46,8 @@ def test_sort_list_of_dicts_by_attribute(self):
         assert [4, 2, 1, 3] == [data["id"] for data in output_list]
         output_list = sort_list_of_dicts_by_attribute(input_list, "price", reverse_order=True)
         assert [3, 1, 2, 4] == [data["id"] for data in output_list]
+
+    def test_convert_datetime_to_ui_str_format(self):
+        current_time = datetime.now()
+        current_time_str = convert_datetime_to_ui_str_format(current_time)
+        assert type(current_time_str) is str
```

Comment:
Sorry for noticing this now: why don't we test the actual output? I think it would be good to pass a specific date and test the full output, not just the type.